api/urls.py

```diff
@@ -22,6 +22,9 @@
     path("", views.landing_page, name="landing"),
     path("admin/", admin.site.urls),
     path("account/", include("accounts.urls")),
+    path('registration/', include('registration.urls')),
+    path('events/', include('events.urls')),
+    path('team/', include('team.urls')),
```

Comment:
why are these two routes required @gaurKrishna

Reply:
Yes those two routes are not required as they have been included through simple router.